molecule_vagrant/modules/vagrant.py

```diff
@@ -589,48 +410,33 @@ def _get_stderr_log(self):
         return self._get_vagrant_log("err")
 
     def _get_vagrant_log(self, __type):
-        instance_name = self._module.params["instance_name"]
-
         return os.path.join(
-            self._config.scenario.ephemeral_directory,
-            "vagrant-{}.{}".format(instance_name, __type),
+            self._config.scenario.ephemeral_directory, "vagrant.{}".format(__type),
         )
 
 
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            instance_name=dict(type="str", required=True),
-            instance_interfaces=dict(type="list", default=[]),
-            instance_raw_config_args=dict(type="list", default=None),
-            config_options=dict(type="dict", default={}),
-            platform_box=dict(type="str", required=False),
-            platform_box_version=dict(type="str"),
-            platform_box_url=dict(type="str"),
-            provider_name=dict(type="str", default="virtualbox"),
-            provider_memory=dict(type="int", default=512),
-            provider_cpus=dict(type="int", default=2),
-            provider_options=dict(type="dict", default={}),
-            provider_override_args=dict(type="list", default=None),
-            provider_raw_config_args=dict(type="list", default=None),
+            instances=dict(type="list", required=True),
+            provider=dict(type="str", default="virtualbox"),
```

Comment:
In a perfect world the parameters should have remained identical but I think it's not possible at all.
So, maybe, the only thing to do is to document somewhere that things have changed between 0.3 and 0.4 to avoid bugs from people with their own ``create.yml`` / ``destroy.yml`` playbooks.

Reply:
Added a note about it in 79db427.